vital/algo/detected_object_set_output.h

```diff
@@ -35,28 +38,35 @@ namespace algo {
  *
  */
 class VITAL_ALGO_EXPORT detected_object_set_output
-  : public kwiver::vital::algorithm_def<detected_object_set_output>
+  : public kwiver::vital::algorithm_def< detected_object_set_output >
 {
 public:
   virtual ~detected_object_set_output();
 
   /// Return the name of this algorithm
-  static std::string static_type_name() { return "detected_object_set_output"; }
+  static std::string
+  static_type_name()
+  {
+    return "detected_object_set_output";
+  }
```

Comment:
Not the end of the world, but one of these days I'd really like to figure out why Uncrustify isn't permitting one-liners. Supposedly it is configured to do so.

Reply:
In this specific case, the "one liner" is ruled out by the constraint of 79 chars/line.  As a one-liner, this line comes to 80 characters, so Uncrustify is re-writing it.  I tested it with a setting of 80 chars/line and the one-line is retained.